.github/workflows/test-chart.yml

```diff
@@ -0,0 +1,102 @@
+name: Test Chart
+
+on:
+  push:
+  pull_request:
+
+env:
+  # The actual version will be injected to the chart by the task that builds the chart
+  DUMMY_VERSION: 1.2.3
+
+jobs:
+  test-e2e:
+    name: Run on Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone the code
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: Install the latest version of kind
+        run: |
+          curl -Lo ./kind https://kind.sigs.k8s.io/dl/latest/kind-linux-amd64
+          chmod +x ./kind
+          sudo mv ./kind /usr/local/bin/kind
+
+      - name: Verify kind installation
+        run: kind version
+
+      - name: Create kind cluster
+        run: kind create cluster
```

Comment:
Are we using default config here? What specifications is the cluster?

Reply:
Yes. It's using the default Kind configs.
I not sure what are the defaults, and I'm not sure it's properly documented. [The documentation](https://kind.sigs.k8s.io/docs/user/configuration/) mentions a few words about defaults, but I could not find a summary of all defaults (The workflow, including the command for creating the Kind cluster was auto-generated by kubebuilder).

Reply:
Do we maybe want to use predefined config?

Reply:
I think for now we should be good without it. If later on we see a need, we can have that added.